src/server/accounting.c

```diff
@@ -2297,21 +2298,32 @@ void
 log_suspend_resume_record(job *pjob, char *user, char *host, int acct_type)
 {
 	char *buf = NULL;
-	char *tmp;
 	char *resc_used;
+	char *resc_released;
+	int buf_size = 1024;
 	int resc_used_size = 0;
-	int rc;
 
-	rc = pbs_asprintf(&buf, "requestor=%s@%s", user, host);
+	buf = malloc(buf_size * sizeof(char));
+	if (buf == NULL)
+		return;
+
+	buf[0] = '\0';
+
+	snprintf(buf, buf_size, "requestor=%s@%s", user, host);
 
 	if (acct_type == PBS_ACCT_SUSPEND) {
 		
 		if (pjob->ji_wattr[JOB_ATR_resc_released].at_flags & ATR_VFLAG_SET) {
-			tmp = buf;
-			rc = pbs_asprintf(&buf, "%s resources_released=%s", tmp,
-			pjob->ji_wattr[JOB_ATR_resc_released].at_val.at_str);
-			free(tmp);
-		}	
+			if (pbs_asprintf(&resc_released, " resources_released=%s",
```

Comment:
Consider doing two calls to pbs_strcat() instead of calling pbs_asprintf().  It will allocate memory which is not necessary.  You are only concatenating two strings together.  If it was more than that, pbs_asprintf() would be better.  In this case, I think the two calls to pbs_strcat() is better.

Reply:
Hi @bhroam, there is a change for removal of requestor attribute from the log records. Also, I have addressed your above comment in the latest commit. Please have a look at it.